sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -292,7 +293,7 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
             ScaleBarPluginImpl()
           }
           else -> {
-            plugin.instance ?: throw RuntimeException("Custom non Mapbox plugins must have non-null `instance` parameter!")
+            plugin.instance ?: throw MapboxMapException("Custom non Mapbox plugins must have non-null `instance` parameter!")
```

Comment:
shall we have a dedicated Plugin exception to indicate the plugin setup issue?

Reply:
yeah, we might even already have one for that, I'll check